test/readers/iso19115_2/tc_iso19115_2_extent.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# MdTranslator - minitest of
+# readers / iso19115-2 / module_extent
+
+require 'adiwg/mdtranslator/readers/iso19115_2/modules/module_extent'
+require_relative 'iso19115_2_test_parent'
+
+class TestReaderIso191152Extent < TestReaderIso191152Parent
+   @@nameSpace = ADIWG::Mdtranslator::Readers::Iso191152::Extent
+
+   def test_extent_complete 
+      xDoc = TestReaderIso191152Parent.get_xml('iso19115-2.xml')
+      TestReaderIso191152Parent.set_xdoc(xDoc)
+
+      xIn = xDoc.xpath('.//gmd:extent')[0]
+      hResponse = Marshal.load(Marshal.dump(@@hResponseObj))
+      hDictionary = @@nameSpace.unpack(xIn, hResponse)
+
+      refute_empty hDictionary
+      assert hDictionary.instance_of? Hash
+
+      hGeoExtent = hDictionary[:geographicExtents][0]
+      refute_empty hGeoExtent[:boundingBox]
+      assert_equal("true", hGeoExtent[:containsData])
```

Comment:
just check that `hDictionary[:geographicExtents]` is of particular size (i.e. `assert_equal(1, hDictionary[:geographicExtents].size)` . add a geo extent test module and have it be responsible for its own content. typically we want the child to be responsible for asserting its own content

Reply:
a module which contains other modules ( e.g. `extent` contains an array of `:geographicExtents` ) should simply check the size of that array not the content of the elements in the array itself

Reply:
added all above tests, please review again. Thanks!